spec/swagger.yaml

```diff
@@ -10031,3 +10062,144 @@ definitions:
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          required:
+            - lastUpdatedDate
+            - agency
+            - translationJobs
+          properties:
+            lastUpdatedDate:
+              type: string
+              format: date-time
+              description: Date when data for the api response was retrieved.
+            agency:
+              $ref: '#/definitions/VendorsAPIJobsListAgencyItem'
+            translationJobs:
+              type: array
+              items:
+                $ref: '#/definitions/VendorsAPIJobsListTranslationJobItem'
+
+  VendorsAPIJobsListAgencyItem:
+    type: object
+    required:
+      - agencyUid
+      - agencyName
+    properties:
+      agencyUid:
+        type: string
+      agencyName:
+        type: string
+
+  VendorsAPIJobsListTranslationJobItem:
+    type: object
+    required:
+      - account
+      - project
+      - progressDetails
+      - translationJob
+    properties:
+      account:
+        type: object
+        properties:
+          accountName:
+            type: string
+          accountUid:
+            type: string
+      project:
+        type: object
+        properties:
+          projectId:
+            type: string
+          projectName:
+            type: string
+      progressDetails:
+        $ref: '#/definitions/VendorsAPIJobsListAgencyItemProgressDetailsItem'
+      translationJob:
+        $ref: '#/definitions/VendorsAPIJobsListAgencyItemTranslationJobItem'
+
+  VendorsAPIJobsListAgencyItemProgressDetailsItem:
+    type: object
+    properties:
+      aggregatedPerWorkflow:
+        type: object
+        properties:
+          workflows:
+            type: array
+            items:
+              $ref: '#/definitions/VendorsAPIJobsListAgencyItemProgressDetailsItemWorkflowsItem'
+
+  VendorsAPIJobsListAgencyItemProgressDetailsItemWorkflowsItem:
+    type: object
+    properties:
+      workflowUid:
+        type: string
+      workflowName:
+        type: string
+      targetLocales:
+        type: array
+        items:
+          $ref: '#/definitions/VendorsAPIJobsListAgencyItemProgressDetailsItemWorkflowsItemTargetLocalesItem'
+
+  VendorsAPIJobsListAgencyItemProgressDetailsItemWorkflowsItemTargetLocalesItem:
+    type: object
+    properties:
+      localeId:
+        type: string
+      issues:
+        type: object
+        properties:
+          totalIssuesCount:
+            type: integer
+      workflowSteps:
+        $ref: '#/definitions/VendorsAPIJobsListAgencyItemProgressDetailsItemWorkflowsItemTargetLocalesItemWorkflowStepsItem'
+
+  VendorsAPIJobsListAgencyItemProgressDetailsItemWorkflowsItemTargetLocalesItemWorkflowStepsItem:
+    type: object
+    properties:
+      wordCount:
+        type: integer
+      unassignedWordCount:
+        type: integer
+      workflowStepName:
+        type: string
+      workflowStepOrder:
```

Comment:
based on upcoming changes with workflows, workflowStepOrder may be something we want to hold off on. @klegins-smartling?